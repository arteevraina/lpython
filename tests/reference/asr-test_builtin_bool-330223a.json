{
    "basename": "asr-test_builtin_bool-330223a",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/test_builtin_bool.py",
    "infile_hash": "3a740faba0e36b55c179ab0fa08486dc3080332e79c8e38b8f25b4a4",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_builtin_bool-330223a.stderr",
    "stderr_hash": "5cf42029c7abd7dc2dedc535fff29c622c3871ec342b595c426b27f3",
    "returncode": 2
}