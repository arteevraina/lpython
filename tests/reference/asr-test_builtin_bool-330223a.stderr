warning: The module 'lpython_builtin' located in $DIR/src/bin/../runtime/lpython_builtin.py cannot be loaded
  --> tests/../integration_tests/test_builtin_bool.py:56:13
   |
56 |     c = c32(complex(2, 3))
   |             ^^^^^^^^^^^^^ imported here

semantic error: The symbol '_lpython_floordiv' not found in the module 'lpython_builtin'
   --> $DIR/src/bin/../runtime/lpython_builtin.py:209:15
    |
209 |     if (n_ - (n_ // 2)*2) == 0:
    |               ^^^^^^^ 
